### DISCUSSION TITLE
課題07についての質問

### DISCUSSION THREAD

**Discussion by TakedaAyano** [nsc-kiwamu/traning-takeda#21] [Q&A]
https://github.com/nsc-kiwamu/traning-takeda/pull/20#issue-1298310059

↑上記のリンク先に質問を書かせていただいております。
よろしくお願いいたします。

**Comment by nsc-kiwamu** [comment]
https://github.com/nsc-kiwamu/traning-takeda/pull/20#issuecomment-1179539009
